dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -1,41 +1,55 @@
 -# English only for now
 - if language == "en"
```

Comment:
or put it in SurveyResultHelper and include that in ApplicationHelper. 

It might also make sense to move the partial from views/layouts to views/home if it only goes on the homepage